aptos-move/framework/aptos-stdlib/sources/comparator.move

```diff
@@ -82,23 +85,16 @@ module aptos_std::comparator {
     }
 
     #[test]
+    #[expected_failure]
```

Comment:
Why not create a different test that catches the issue found here? Modifying a test that verifies other cases that still work reduces test coverage.

Reply:
Ok, I will fix this.